lib/recovery/recover_lost_jobs.rb

```diff
@@ -12,14 +12,31 @@ class RecoverLostJobs
     ].freeze
 
     def self.run
+      requeued_count = 0
+      skipped_count = 0
+
       STRATEGIES.each do |strategy|
         model = strategy[:model]
         worker = strategy[:worker]
 
         model.unprocessed.where("created_at <= ?", 1.hour.ago).map do |record|
-          worker.perform_async(record.id)
+          formatted_record = "{record.class}: #{record.slice(:id, :created_at)}"
```

Comment:
We shouldn't need this logging, except perhaps for the "success" one: the config for the unique jobs gem means we'll log if the job is a no-op. (And I don't think we have a need for knowing the number of recovered jobs.)